### PR TITLE
Support for django rest framework

### PR DESCRIPTION
**Pull request by LegoStormtroopr**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
           - 13306:3306
         # Set health checks to wait until postgres has started
```
Comment (s-i-l-k-e) [inline .github/workflows/ci.yml:35]: Comment says Postgres, we're pinging MySQL, and running the tests on Postgres and SQLite?

File: README.md [hunk 2]
```diff
@@ -103,11 +103,13 @@ Tested on:
 
 
 Pros:
-* Fetching all translations for a models requires a single query
+* Battletested in production - [Aristotle Metadata](https://www.aristotlemetadata.com) built, support and uses this library for 2 separate products, served to government and enterprise clients!
+* Fetching all translations for a model requires a single query
 * Translations are stored in a single database field with the model
-* Translations act like regular a field `Model.field_name = "some string"` and `print(Model.field_name)` work as you would expect
+* All translations act like regular a field `Model.field_name = "some string"` and `print(Model.field_name)` work as you would expect
```
Comment (s-i-l-k-e) [inline README.md:109]:
```suggestion
* All translations act like a regular field`Model.field_name = "some string"` and `print(Model.field_name)` work as you would expect
```

File: garnett/drf.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            return self.innerfield.validators
+
+        return self._validators
+
+    @validators.setter
+    def validators(self, validators):
+        self._validators = validators
+
+    def run_validation(self, data=empty):
+        (is_empty_value, data) = self.validate_empty_values(data)
```
Comment (s-i-l-k-e) [inline garnett/drf.py:37]: Brackets aren't necessary, but also I don't think they are particularly harmful

File: garnett/drf.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            return data
+        value = self.to_internal_value(data)
+
+        self.translation_type = None
+        if isinstance(data, str):
+            self.translation_type = LanguageTypes.monolingual
+        elif isinstance(data, dict):
+            self.translation_type = LanguageTypes.multilingual
+        else:
+            print("Unknown attached")
```
Comment (s-i-l-k-e) [inline garnett/drf.py:48]: Not really sure what this print statement means

**Review by s-i-l-k-e — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (LegoStormtroopr) [inline .github/workflows/ci.yml:35]:
```suggestion
        # Set health checks to wait until mariadb has started
```

File: garnett/drf.py (hunk 3 above)
Comment (LegoStormtroopr) [reply garnett/drf.py:37]: This was lifted straight from django-rest-framework

**Pull request merged**